layouts/error.vue

```diff
@@ -220,6 +219,7 @@
         flex-direction: column;
         justify-content: center;
         align-items: center;
+        overflow: hiddem;
```

Comment:
This should be `overflow: hidden`. Besides, it has no effect on the scroll bar because it is targeting the wrong container.

Reply:
oops
there was a scrollbar in the cat popup and i wanted to remove that